public/app/plugins/datasource/loki/querybuilder/components/LokiQueryBuilderOptions.tsx

```diff
@@ -119,6 +131,21 @@ export const LokiQueryBuilderOptions = React.memo<Props>(
               aria-label="Select resolution"
             />
           </EditorField>
+          {config.featureToggles.lokiQuerySplitting && (
+            <EditorField
+              label="Chunk Duration"
+              tooltip="Defines the duration of a single query chunk when query chunking is used."
+            >
+              <AutoSizeInput
+                minWidth={14}
+                type="string"
+                min={0}
+                defaultValue={query.chunkDuration?.toString() ?? '1d'}
```

Comment:
do we need the `.toString()` here? it seems to be already a string...

Reply:
Good catch, changed in a4be4d478f.